sphinx-doc/conf.py

```diff
@@ -19,7 +19,8 @@
 extensions = [
     'nbsphinx', 'sphinx.ext.autodoc', 'sphinx.ext.autosummary',
     'sphinx.ext.napoleon', 'sphinx.ext.intersphinx', 'sphinx.ext.mathjax',
-    'sphinx.ext.todo', 'IPython.sphinxext.ipython_console_highlighting'
+    'sphinx.ext.todo', 'IPython.sphinxext.ipython_console_highlighting',
+    'sphinx.ext.autosectionlabel'
```

Comment:
What does the autosectionlabel extension do?

Reply:
This means you can reference intra-dcoument sections automatically with their names.

Reply:
ok, well maybe it is the cause of all the sphinx build errors then?

```
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/deprecated.rst:5: WARNING: duplicate label v2.x, other instance in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/changelog.rst
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/migrating.rst:82: WARNING: duplicate label components, other instance in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/components.rst
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/style.rst:78: WARNING: duplicate label tools, other instance in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/style.rst
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/style.rst:87: WARNING: duplicate label documentation, other instance in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/style.rst
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/tutorial/00-Introducing-HOOMD-blue/02-Performing-Hard-Particle-Monte-Carlo-Simulations.ipynb:12: WARNING: duplicate label overview, other instance in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/tutorial/00-Introducing-HOOMD-blue/01-The-Simulation-Object.ipynb
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/tutorial/00-Introducing-HOOMD-blue/02-Performing-Hard-Particle-Monte-Carlo-Simulations.ipynb:15: WARNING: duplicate label questions, other instance in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/tutorial/00-Introducing-HOOMD-blue/01-The-Simulation-Object.ipynb
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/tutorial/00-Introducing-HOOMD-blue/02-Performing-Hard-Particle-Monte-Carlo-Simulations.ipynb:21: WARNING: duplicate label objectives, other instance in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-blue/checkouts/1011/sphinx-doc/tutorial/00-Introducing-HOOMD-blue/01-The-Simulation-Object.ipynb
...```

We don't use unique header names across the whole codebase.

Reply:
I don't know how that plugin works. I checked the docs and they are rather sparse.